### PR TITLE
Imported scenario keeps the status / scheduling

### PR DESCRIPTION
**Pull request by RomuDeuxfois**
### Related issues

* https://github.com/OpenBAS-Platform/openbas/issues/903

### Testing

Please, test different export/import with teams, players and variables to validate all workflows.

### CODE REVIEW

File: openbas-api/src/main/java/io/openbas/service/ScenarioService.java
Comment (Christian-DONGMO) [inline openbas-api/src/main/java/io/openbas/service/ScenarioService.java:1]: You should also split the exportScenario method into smaller methods

**Review by Christian-DONGMO — CHANGES_REQUESTED** [review]

File: openbas-api/src/main/java/io/openbas/importer/V1_DataImporter.java
[the remark this answers is not in the document]
Comment (RomuDeuxfois) [reply openbas-api/src/main/java/io/openbas/importer/V1_DataImporter.java:1]: I dit this part

File: openbas-api/src/main/java/io/openbas/service/ScenarioService.java
Comment (RomuDeuxfois) [reply openbas-api/src/main/java/io/openbas/service/ScenarioService.java:1]: It will be in another step

**Review by Christian-DONGMO — APPROVED** [review]

**Comment by Christian-DONGMO** [comment]
Nice, Just remember to rebase

**Pull request merged**